independent-projects/arc/runtime/src/main/java/io/quarkus/arc/AlternativePriority.java

```diff
@@ -10,10 +10,10 @@
 /**
  * If a bean is annotated with this annotation, it is considered an enabled alternative with given priority.
  * Effectively, this is a shortcut for {@code Alternative} plus {@code Priority} annotations.
+ * <p>
+ * This annotation can be used not only on bean classes, but also method and field producers.
  *
- * This annotation can be used not only on bean classes, but also method and field producers (unlike pure {@code Priority}).
- *
- * @deprecated Use {@link Alternative} and {@link io.quarkus.arc.Priority}/{@link jakarta.annotation.Priority} instead
+ * @deprecated Use {@link Alternative} and {@link jakarta.annotation.Priority} instead
```

Comment:
I wonder if we should remove this annotation instead...

Reply:
I would be in favor, but ... should we first add a warning that it's deprecated for removal?

Reply:
Well, it's deprecated since 2.6.0.CR1...

Reply:
+1 for removal, it was meant as a temporary solution until we can make use of a standard solution anyway.

Reply:
If it has been deprecated since 2.6, we can remove it. Just make sure you had an entry to the migration guide about how it should be transformed.